.github/workflows/test_backend.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         run: poetry run uvicorn test_observer.main:app --host 0.0.0.0 --port 30000 &
         shell: bash
       - name: Fetch OpenAPI schema
-        run: curl http://localhost:30000/openapi.json -o fetched_openapi.json
+        run: curl http://localhost:30000/openapi.json | jq > fetched_openapi.json
```

Comment:
I added this to avoid enormous diffs on small changes to the API (and it of course improves readability)